third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -121,6 +121,12 @@ func resourceSqlDatabaseInstance() *schema.Resource {
 							Computed:    true,
 							Description: `Used to make sure changes to the settings block are atomic.`,
 						},
+						"deletion_protection": {
```

Comment:
why is this in settings instead of top-level on the resource?

Reply:
That is where the settings are specified for sql instances. I can make it a top level key.